### PR TITLE
Minor Map Fixes: The Delta Gambit

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
I've decided to atomise my minor map fixes PRs going forward by station. First up is Delta- I've given it a good look over with the debugging tools available, and resolved a few minor issues. Some disconnected atmos pipes, some duplicated camera tags, and a couple doubled wires. If there's any issues open that I've missed, please feel free to shout at me to fix them.

## Why It's Good For The Game
Map maintenance is important, and someone's gotta do it.

## Changelog
:cl:
fix: A few little fixes on Deltastation.
/:cl:

### CODE REVIEW

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -97560,7 +97556,6 @@
 	name = "Medbay Maintenance";
 	req_access_txt = "5"
 	},
```
Comment (Timberpoes) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:97558]: Is... This a decal with a name and required access level?

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -33150,18 +33150,14 @@
...
-	dir = 4
+/obj/machinery/camera{
+	c_tag = "Holodeck - Fore 2";
+	name = "holodeck camera"
 	},
-/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden/layer2{
-	dir = 1
+/turf/open/floor/engine{
+	name = "Holodeck Projector Floor"
 	},
```
Comment (Timberpoes) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:33159]: You removed a cable, pipe and scrubber from this turf, but they don't APPEAR to be duplicates. Can you double check this for me? My DMM-reading-fu isn't always the best.

**Review by Timberpoes — CHANGES_REQUESTED** [review]
Map diff bot is neither use nor ornament for this PR.

Matrix-reading the changes via code, it does what it says on the tin. Removes some stuff.

Just a few issues I'm not sure about. It's 7am. I've been up all night. Maybe my mind's playing tricks on me.

**Comment by ArcaneMusic** [comment]
(While you're at it can you hit #53280 for it's instances on delta, since this PR already hits most of the delta map?)

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 2 above)
Comment (EOBGames) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:33159]: I've taken a look over and can't find this. I suspect this is just one of those weird reorganisations you get with .TGM file occasionally.

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 1 above)
Comment (EOBGames) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:97558]: Nah I've just looked and it's just a quirk with the git display, the only thing named "medbay maintenance" is the maint door in medbay.

**Comment by EOBGames** [comment]
> (While you're at it can you hit #53280 for it's instances on delta, since this PR already hits most of the delta map?)

resolved

**Comment by EOBGames** [comment]
if either of those suggestions aren't actually resolved please inform me and I'll look at them again

**Review by Timberpoes — APPROVED** [review]

**Comment by Timberpoes** [comment]
We all gucci in the hucci.

**Comment by Timberpoes** [comment]
If we need any more minor Delta tweaks, we can hit them up in a new PR. No point leaving this hanging for such tiny changes. Greenlit for go!

**Pull request merged**